arrays/c-or-cpp/unique-elements-in-an-array.c

```diff
@@ -0,0 +1,34 @@
+#include<stdio.h>
+
+int main(){
+    int a[1000];
+    int size;
+    printf("Enter size of the array: ");
+    scanf("%d", &size);
+    
+    printf("Enter elements of the array: ");
+    for(int k=0; k<size; k++){
+        scanf("%d", &a[k]);
+    }
+    int flag;
+
+    for(int i=0; i<7; i++){
```

Comment:
You changed the input to work with array of size upto 1000 but your loops will only perform 7 iterations each no matter the size of the array.